src/index.ts

```diff
@@ -22,11 +22,11 @@ const Banal = {
 
     /* CHECKS */
 
-    if ( !options.module?.length ) throw new Error ( 'You need to specify at least one module to analyze' );
+    if ( !options.module?.length && !options.metafile ) throw new Error ( 'You need to specify at least one module to analyze (or explicitly provide a metafile via --metafile)' );
 
     /* PATHS */
 
-    const distPath = dirname ( import.meta.url );
+    const distPath = fileURLToPath(new URL('.', import.meta.url))
```

Comment:
This might be leftover from me trying to make it work on Windows. Have to try again if it now works without, now that `tiny-dirname` is fixed.